packit_service/worker/helpers/build/build_helper.py

```diff
@@ -754,3 +755,22 @@ def report_status_to_configured_job(
             links_to_external_services=links_to_external_services,
             update_feedback_time=update_feedback_time,
         )
+
+    def notify_about_failure_if_configured(self):
+        """
+        If there is a failure_comment_message configured for the job,
+        post a comment and include the configured message. Do not post
+        the comment if the last comment from the Packit user is identical.
+        """
+        if (
+            configured_message := self.job_config.notifications.failure_comment_message
+        ) is None:
+            return
+
+        full_message = (
+            f":exclamation: **One of the {self.comment_job_name} failed for commit "
+            f"{self.db_project_event.commit_sha}**\n\n---\n\n{configured_message}"
+        )
+        self.status_reporter.comment(
+            full_message, duplicate_check=DuplicateCheckMode.check_last_comment
```

Comment:
Thinking about that now, I am slightly worried people would like to define the strategy themselves => not sure if it's needed but we need to be prepared with the config structure to support this if needed. (Yes, I know the config PR is already merged..;(

Reply:
actually, I was thinking about it too and there is no problem with changing the schema since it is not added to the docs or anywhere yet :) ..I was thinking about changing the schema like:
```yaml
notifications:
  failure_comment:
    message:
    duplicate_check:
```

on the other hand, I am not sure whether we are not overcomplicating this...would we then allow one to one mapping to the [DuplicateCheckMode](https://github.com/packit/packit-service/blob/39bc4f08f13450a6b7abfb537ec4e9d367e8dc57/packit_service/worker/reporting.py#L68)?

Reply:
> but we can definitely at least change the schema for message to notifications.failure_comment.message so that the configuration for failure_comment can be easily extended if needed

+1

> would we then allow one to one mapping to the [DuplicateCheckMode](https://github.com/packit/packit-service/blob/39bc4f08f13450a6b7abfb537ec4e9d367e8dc57/packit_service/worker/reporting.py#L68)?

This makes sense, but I would first wait for the users to request this.

Reply:
ok :+1:

opened https://github.com/packit/packit/pull/2075